charts/loki-stack/requirements.yaml

```diff
@@ -6,7 +6,7 @@ dependencies:
 - name: "promtail"
   condition: promtail.enabled
   repository: "https://grafana.github.io/helm-charts"
-  version: "^2.2.0"
+  version: "^2.4.2"
```

Comment:
Perhaps the [error in pipeline](https://github.com/grafana/helm-charts/runs/5473495275?check_suite_focus=true#step:7:31) triggered by this?
Looking at the releases, there seems to be [no Chart with version 2.4.2](https://github.com/grafana/helm-charts/releases?expanded=true&page=3&q=promtail).